src/main/java/org/apache/sysml/parser/DMLTranslator.java

```diff
@@ -1463,12 +1462,11 @@ else if (current instanceof MultiAssignmentStatement) {
 					FunctionType ftype = fsb.getFunctionOpType();
 					FunctionOp fcall = new FunctionOp(ftype, fci.getNamespace(), fci.getName(), finputs, foutputs, false);
 					output.add(fcall);
-					
-					//TODO function output dataops (phase 3)
```

Comment:
One of the crucial changes we need to make in this part, is the extension of DMLTranslator. In detail, this means we need to extend `DMLTranslator.processExpression` to create FunctionOps whenever we encounter a FunctionCallIdentifier (or potentially an unsupported BuiltinFunctionExpression depending how the parser reacts to these function calls in expressions).